android/engine/build.gradle

```diff
@@ -202,7 +202,7 @@ dependencies {
     api('javax.xml.stream:stax-api:1.0')
     api('com.github.ben-manes.caffeine:caffeine:2.9.1')
     
-    api('org.smartregister:data-capture:0.1.0-beta06-preview7-SNAPSHOT') {
+    api('org.smartregister:data-capture:1.0.0-preview-SNAPSHOT') {
```

Comment:
this should be able to point to the new official release now

Reply:
> this should be able to point to the new official release now

Yeah. Our `preview-SNAPSHOT` version is based on our forked repo which already has all the latest changes from upstream https://github.com/google/android-fhir/releases/tag/sdc-1.0.0